README.md

```diff
@@ -148,6 +148,7 @@ var skPromise = Smooch.init({
         messengerChannelDescription: 'Connect your Facebook Messenger account to be notified when you get a reply and carry the conversation on Facebook Messenger.',
         frontendEmailChannelDescription: 'To talk to us using email just send a message to our email address and we\'ll reply shortly:',
         smsChannelDescription: 'To talk to us using SMS, just send a text message to this number from your favorite SMS app:',
+        smsChannelPendingDescription: 'Check your messages at ${number} to confirm your phone number.',
```

Comment:
the convention so far has been just `{}` for variables, not `${}`: https://github.com/smooch/smooch-js/pull/340/files#diff-04c6e90faac2675aa89e2176d2eec7d8L141